aspnetcore/fundamentals/index.md

```diff
@@ -155,55 +164,143 @@ ASP.NET Core uses the [Kestrel](xref:fundamentals/servers/kestrel) server implem
 
-* [Kestrel](xref:fundamentals/servers/kestrel) server is a managed, cross-platform web server. Kestrel is often run in a reverse proxy configuration using [IIS](https://www.iis.net/). Kestrel can also be run as a public-facing edge server exposed directly to the Internet in ASP.NET Core 2.0 or later.
-* [HTTP.sys](xref:fundamentals/servers/httpsys) server is a web server for ASP.NET Core on Windows.
+* *Kestrel* is a cross-platform web server. Kestrel is often run in a reverse proxy configuration using [IIS](https://www.iis.net/). In ASP.NET Core 2.0 or later, Kestrel can be run as a public-facing edge server exposed directly to the Internet.
+* *HTTP.sys* is a server for Windows that isn't used with IIS.
 
 # [macOS](#tab/macos)
 
-ASP.NET Core uses the [Kestrel](xref:fundamentals/servers/kestrel) server implementation. Kestrel is a managed, cross-platform web server. Kestrel can also be run as a public-facing edge server exposed directly to the Internet in ASP.NET Core 2.0 or later.
+ASP.NET Core provides the *Kestrel* cross-platform server implementation. In ASP.NET Core 2.0 or later, Kestrel can be run as a public-facing edge server exposed directly to the Internet.
 
 # [Linux](#tab/linux)
 
-ASP.NET Core uses the [Kestrel](xref:fundamentals/servers/kestrel) server implementation. Kestrel is a managed, cross-platform web server. Kestrel is often run in a reverse proxy configuration with [Nginx](http://nginx.org) or [Apache](https://httpd.apache.org/). Kestrel can also be run as a public-facing edge server exposed directly to the Internet in ASP.NET Core 2.0 or later.
+ASP.NET Core provides the *Kestrel* cross-platform server implementation. In ASP.NET Core 2.0 or later, Kestrel can be run as a public-facing edge server exposed directly to the Internet. Kestrel is often run in a reverse proxy configuration with [Nginx](http://nginx.org) or [Apache](https://httpd.apache.org/).
 
 ---
 
 ::: moniker-end
 
-For more information, see <xref:fundamentals/servers/index>.
+For more information, see [Servers](xref:fundamentals/servers/index).
 
 ## Configuration
 
-ASP.NET Core uses a configuration model based on name-value pairs. The configuration model isn't based on <xref:System.Configuration> or *web.config*. Configuration obtains settings from an ordered set of configuration providers. The built-in configuration providers support a variety of file formats (XML, JSON, INI), environment variables, and command-line arguments. You can also write your own custom configuration providers.
+ASP.NET Core provides a configuration framework that isn't based on [System.Configuration](xref:System.Configuration) or *web.config*. The framework gets settings as name-value pairs from an ordered set of configuration providers. There are built-in configuration providers for a variety of sources, such as *.json* files, *.xml* files, environment variables, and command-line arguments. You can also write custom configuration providers.
```

Comment:
If coming to ASP.NET Core without ASP.NET 4.x experience, this first sentence will be confusing.